apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -410,6 +437,9 @@ k8s-celery-beat:
 		KUMA_CPU_REQUEST=${CELERY_BEAT_CPU_REQUEST} \
 		KUMA_MEMORY_REQUEST=${CELERY_BEAT_MEMORY_REQUEST} \
 		KUMA_ALLOWED_HOSTS="" \
+		NEW_RELIC_APP_NAME=${NEW_RELIC_CELERYBEAT_NAME} \
+		NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
+		NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```

Comment:
I think these two lines:
```
NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```
are not needed since they will be picked up from the existing exported env variables?